src/App.jsx

```diff
@@ -14,45 +15,112 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_FIELD = {
+  SORT_ALPHABETICALLY: 'Sort alphabetically',
+  SORT_BY_LENGTH: 'Sort by length',
+  REVERSE: 'Reverse',
+};
+
+export const App = () => {
+  const [arrayValues, setArrayValues] = useState(goodsFromServer);
```

Comment:
no need to save all goods in state, also names are not meaningful, we should avoid general words in naming as 'array, obj, item, value, etc.'
remove this state and make function which will return filtered goods in accordance with all filters